kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SetUpLearningFacilityForm.vue

```diff
@@ -19,6 +19,8 @@
     />
     <SelectDeviceModalGroup
       v-if="showSelectAddressModal"
+      :filterByFacilityCanSignUp="true"
```

Comment:
Do we need to filter by this? We should be able to import facilities even if they don't allow sign up, right?

Reply:
nope, it was a unneeded code I left when testing it, but deleted it in another commit a few seconds after this one. If you look at the complete PR code this line is not there  :wink: